test/functional/apps/dashboard/group5/data_shared_attributes.ts

```diff
@@ -74,7 +75,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     it('data-shared-item title should update a viz when using a custom panel title', async () => {
       await PageObjects.dashboard.switchToEditMode();
       const CUSTOM_VIS_TITLE = 'ima custom title for a vis!';
-      await dashboardPanelActions.setCustomPanelTitle(CUSTOM_VIS_TITLE);
+      await dashboardPanelActions.customizePanel();
+      await dashboardCustomizePanel.expectCustomizePanelSettingsFlyoutOpen();
+      await dashboardCustomizePanel.setCustomPanelTitle(CUSTOM_VIS_TITLE);
+      await dashboardCustomizePanel.clickFlyoutPrimaryButton();
```

Comment:
Maybe this function should have a more explicit name? Like `.clickSaveButton()` or whatever the button actually does.